src/app/hero/hero.component.ts

```diff
@@ -9,7 +9,7 @@ export class HeroComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit() {
+  ngOnInit() : void {
```

Comment:
@qualmless would you mind remove the space before the colon? I'm seeing a new lint error from the white space. You can run `npm run lint` and check that the output doesn't contain any lint error from _hero.component.ts_. Thanks!